src/main/java/com/employeesService/EmployeesService/model/DepartmentEmployee.java

```diff
@@ -0,0 +1,69 @@
+package com.employeesService.EmployeesService.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "dept_emp")
+@IdClass(DepartmentEmployee.DepartmentEmployeeId.class)
+public class DepartmentEmployee {
```

Comment:
I expected to design it like this: https://vladmihalcea.com/the-best-way-to-map-a-many-to-many-association-with-extra-columns-when-using-jpa-and-hibernate/

Try to change it like that above (link) ^^